share/spice/rfc/footer.handlebars

```diff
@@ -0,0 +1,3 @@
+<div class="one-line">
+    <div class="h-xs tx-clr--slate">RFC {{number}}</div>
```

Comment:
Why don't you just use the `title` and `subtitle` attributes of the `text` template?